components/vision.py

```diff
@@ -223,6 +227,8 @@ def execute(self) -> None:
             self.last_recieved_timestep = time.monotonic()
             self.last_timestamp = timestamp
```

Comment:
Have we double checked what time base these timestamps use? At a quick glance to me it looks like photonlibpy uses wpilib.Timer.getFPGATimestamp().

Reply:
Nice one